plugins/inputs/ethtool/namespace_linux.go

```diff
@@ -103,7 +103,7 @@ func (n *NamespaceGoroutine) Start() error {
 		// Every namespace needs its own connection to ethtool
 		e, err := ethtoolLib.NewEthtool()
 		if err != nil {
-			n.Log.Errorf(`Could not create ethtool client for namespace "%s": %s`, n.name, err)
+			n.Log.Errorf("Could not create ethtool client for namespace %q: %w", n.name, err)
```

Comment:
`%w` should be only used for creating error, not for printing. Use `%s` or `%v` instead.